aten/src/ATen/detail/XPUHooksInterface.h

```diff
@@ -14,6 +13,10 @@ namespace at {
 class Context;
 }
 
+// We use forward declaration here instead of #include <ATen/dlpack.h> to avoid
+// Meta-internal DLPack version conflict.
```

Comment:
```suggestion
// leaking DLPack implementation detail to every project that includes `ATen/Context.h`, which in turn
// would lead to a conflict when linked with another project using DLPack (for example TVM)
```